### ISSUE TITLE
"Game is not installed" when launching Heroic in offline mode

### ISSUE THREAD

**Issue by OlegAckbar** [Heroic-Games-Launcher/HeroicGamesLauncher#2606]
### Describe the bug

When I'm opening Heroic without internet connection it says "Game is not installed" for my Borderlands 3 installation yet it is installed. But it looks like it installed and I can run it when pressing "Play" button, yet trying to launch it through Steam shortcut will throw error "Game is not installed".
If I'll turn internet connection off when Heroic already started no such problem observed. If I'll turn internet connection on after launching Heroic in offline mode it will still show that games is not installed.

### Add logs

```text
(20:16:21) INFO:    [Connection]:       Connectivity: offline
(20:16:21) INFO:    [Legendary]:        Running command: /opt/Heroic/resources/app.asar.unpacked/build/bin/linux/legendary --version
(20:16:21) INFO:    [Legendary]:        Legendary location: /opt/Heroic/resources/app.asar.unpacked/build/bin/linux/legendary
(20:16:21) INFO:    [Gog]:              GOGDL location: /opt/Heroic/resources/app.asar.unpacked/build/bin/linux/gogdl
(20:16:21) INFO:    [Gog]:              GOGDL location: /opt/Heroic/resources/app.asar.unpacked/build/bin/linux/gogdl
(20:16:21) WARNING: [Backend]:          Protocol already registered.
(20:16:21) INFO:    [Gog]:              Running command: /opt/Heroic/resources/app.asar.unpacked/build/bin/linux/gogdl --auth-config-path /home/deck/.config/heroic/gog_store/auth.json --version
(20:16:22) INFO:    [Backend]:          Loading Screen Ready
(20:16:22) INFO:    [Frontend]:         Refreshing Library
(20:16:22) INFO:    [Legendary]:        Refreshing library...
(20:16:22) INFO:    [Legendary]:        Found 0 games to update
(20:16:22) INFO:    [Backend]:          Frontend Ready
(20:16:22) INFO:    [Backend]:          Checking for new Heroic Updates
(20:16:22) ERROR:   [Backend]:          Error when checking for Heroic updates Error: getaddrinfo EAI_AGAIN api.github.com
    at GetAddrInfoReqWrap.onlookup [as oncomplete] (node:dns:107:26)
(20:16:22) INFO:    [Backend]:          Checking for current version changelog
(20:16:22) ERROR:   [Backend]:          Error when checking for current Heroic changelog
(20:16:22) INFO:    [Backend]:          

Heroic Version: 2.7.0 Eustass Kid
Legendary Version:  0.20.32 Dark Energy (hotfix #6)
GOGdl Version: 0.7.1

Electron Version: 23.1.4
Chrome Version: 110.0.5481.192
NodeJS Version: 18.12.1

OS: SteamOS KERNEL: 5.13.0-valve36-1-neptune ARCH: x64
CPU: AMD Custom APU 0405 @2.17 GOVERNOR: schedutil
RAM: Total: 11.52 GiB Available: 7.83 GiB
GRAPHICS: GPU0: VanGogh [AMD Custom GPU 0405] VRAM: 512MB 
PROTOCOL: x11

(20:16:24) WARNING: [DXVKInstaller]:    App offline, skipping possible DXVK update.
(20:16:27) INFO:    [Backend]:          Starting the Download Queue
```

### Steps to reproduce

1. Install Borderlands 3
2. Close Heroic
3. Turn off internet connection
4. Launch Heroic
5. See "Game is not installed" in library
6. Try to launch it from Steam shortcut in that state, it will throw error "Game is not installed"

### Expected behavior

I want to play this game when my Steam Deck has no internet connection. As I can see it's pretty much possible but some strange Heroic bug won't allow me to run this game if it was started in offline mode, yet it CAN start it anyway, so there's no problem on game's side.

### Screenshots

![Screenshot_20230406_202044](https://user-images.githubusercontent.com/9915567/230451500-a31f51bc-1e43-49d8-b617-d09c42330d79.png)

### Heroic Version

Latest Stable

### System Information

- OS: SteamOS 3.4.6
- Heroic version: manually installed .pacman package of Heroic 2.7.0

**Comment by tutuho** [comment]
I have the same error, while online I keep getting the same error, I couldn't play any game. I have this error since I upgraded to 2.7.0

**Comment by FoodFrisbee** [comment]
I have a "workaround" solution to this issue, select install, select import game, navigate to the game folder then select chosse, and you can play the game. I have to do this everytime I want to play any game via the heroic games launcher

**Comment by 31n70pf** [comment]
For me it is not only with epic games but the whole library (epic and gog) that doesn't load while offline. I can't start anything from the steam overlay. In desktop mode heroic doesn't show any games. I also noticed it after the update to 2.7.

**Comment by djibux** [comment]
While waiting for this bug to be fixed I reverted to version 2.6.1 which works.  I needed a way to play offline.

```
sudo flatpak update --commit=e005060fbf1d5591433b4a33d3b2cdb6acbd4195b08f283563c9494d7f66959b com.heroicgameslauncher.hgl
```

**Comment by flavioislima** [comment]
We have just released a hotfix, will soon be available on Flathub as well.
Closing this one
